_static/api-examples.md

```diff
@@ -126,7 +126,7 @@ You must authenticate with the CDX CROMERR service for signing a manifest. Value
 ```bash
 curl -i -X POST -H 'Content-Type: application/json' \
   --data @auth-creds.json \
-  https://e-manifest.18f.gov/api/0.1/user/authenticate
+  https://e-manifest.18f.gov/api/v1/users/authentications
```

Comment:
I thought about re-naming `authenticate` to `sessions` -- but was worried that was too far from what we have already. 
What do you think of that?

The reason for plurals is that it is the Rails-y way: http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions

But we can certainly go back to singular if preferred. eg: I kept `GET manifest` singular because the path is just `manifest` rather than `manifest/:id` and I didn't want people to think that was an endpoint for a collection. Not against exceptions to the Rails rule so we can stray from it wherever it makes sense.